pkg/pgmodel/sql_bench_test.go

```diff
@@ -0,0 +1,262 @@
+package pgmodel
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/jackc/pgx/v4/pgxpool"
+)
+
+const (
+	labelCount      = 5
+	metricName      = "benchmark_metric"
+	otherMetricName = "other_benchmark_metric"
+)
+
+func BenchmarkGetSeriesIDForKeyValueArrayExistingSeries(b *testing.B) {
+	b.StopTimer()
+	withDB(b, "bench_1", func(db *pgxpool.Pool, t testing.TB) {
+		err := createMetricTableName(db, metricName)
+		if err != nil {
+			t.Fatal(err)
+		}
+
+		keys, values := generateKeysAndValues(labelCount, "label")
```

Comment:
shouldn't this be b.N+labelCount?

Reply:
I thought this test was running the function for an existing series. Hence it only uses one series with 5 labels. Why would it need to be more than that?